static/app/components/modals/metricWidgetViewerModal.tsx

```diff
@@ -25,20 +25,21 @@ import type {Organization} from 'sentry/types';
 import {getDdmUrl, getWidgetTitle, isCustomMetric} from 'sentry/utils/metrics';
 import {emptyMetricsQueryWidget} from 'sentry/utils/metrics/constants';
 import type {MetricQueryWidgetParams, MetricsQuery} from 'sentry/utils/metrics/types';
+import type {MetricsQueryApiRequestQuery} from 'sentry/utils/metrics/useMetricsQuery';
 import useOrganization from 'sentry/utils/useOrganization';
 import usePageFilters from 'sentry/utils/usePageFilters';
 import useRouter from 'sentry/utils/useRouter';
+import {
+  getMetricQueries,
+  toMetricDisplayType,
+} from 'sentry/views/dashboards/metrics/utils';
 import {WidgetDescription} from 'sentry/views/dashboards/widgetCard';
 import {getCreateAlert} from 'sentry/views/ddm/metricQueryContextMenu';
 import {Query} from 'sentry/views/ddm/queries';
 import {MetricDetails} from 'sentry/views/ddm/widgetDetails';
 import {OrganizationContext} from 'sentry/views/organizationContext';
 
-import {
-  convertToDashboardWidget,
-  convertToMetricWidget,
-  toMetricDisplayType,
-} from '../../utils/metrics/dashboard';
+import {convertToDashboardWidget} from '../../utils/metrics/dashboard';
 import {MetricWidget} from '../../views/ddm/widget';
 import {Tooltip} from '../tooltip';
```

Comment:
nitpicking: not sure how, but we ended up with a few relative imports in a few places. 🤔  I find it rather hard to parse as soon as they utilize `../`. I will try to rewrite them whenever I encounter them.

Reply:
I'll take care of these in a followup